### PR TITLE
closes #1692 align python versions on ansible tox

### LINKED ISSUE #1692: align python versions in tox.ini on the supported versions in ansible
# Issue Type

- Bug report

# Desired Behavior

The tested platforms should be aligned on platforms supported by ansible.

# Actual Behaviour

ansible is tested (tox.ini) on py26,py27,py35,py36, 
molecule is tested on py{27,36,37}.

### PR DESCRIPTION
**Pull request by themr0c**
Signed-off-by: Fabrice Flore-Thebault <themr0c@users.noreply.github.com>

closes #1692 align python versions on ansible tox

#### PR Type

- Bugfix Pull Request

### CODE REVIEW

**Comment by themr0c** [comment]
Build error is due to unrelated already present lint errors addressed in another PR.

File: tox.ini [hunk 1]
```diff
@@ -1,8 +1,8 @@
 [tox]
 minversion = 3.7.0
 envlist =
-    py{27,36,37}-ansible{24,25,26,27}-{functional,unit}
-    py{27,36,37}-lint
+    py{26,27,35,36,37}-ansible{24,25,26,27}-{functional,unit}
```
Comment (gundalow) [inline tox.ini]: https://docs.ansible.com/ansible/devel/dev_guide/developing_python_3.html#minimum-version-of-python-3-x-and-python-2-x

> On the controller we support Python 3.5 or greater and Python 2.7 or greater.
> Module-side, we support Python 3.5 or greater and Python 2.6 or greater.

Comment (themr0c) [reply tox.ini]: done my homework

**Comment by themr0c** [comment]
Build errors are not related to the PR: lint issue was already there and this one happens sometime and should maibe some day get a ticket and a fix:
```
fatal: [instance-2]: FAILED! => {"changed": false, "cmd": "/sbin/udevadm info --query property --name /dev/sda1", "msg": "Timer expired after 10 seconds", "rc": 257}
```

File: tox.ini [hunk 2]
```diff
@@ -1,8 +1,8 @@
 [tox]
 minversion = 3.7.0
 envlist =
-    py{27,36,37}-ansible{24,25,26,27}-{functional,unit}
-    py{27,36,37}-lint
+    py{27,35,36,37}-ansible{25,26,27}-{functional,unit}
+    py{27,35,36,37}-lint
```
Comment (webknjaz) [inline tox.ini]: Could you please put it first? Linters are faster so it should be better feedback timing this way

Comment (themr0c) [reply tox.ini]: this was fixed in master

**Comment by themr0c** [comment]
The change refactored and rebased now in a single commit.

**Comment by gundalow** [comment]
Thanks everybody, merged

**Pull request merged**